platform/libs.junit4/nbproject/project.xml

```diff
@@ -26,24 +26,45 @@
             <code-name-base>org.netbeans.libs.junit4</code-name-base>
             <module-dependencies/>
             <public-packages>
+                <package>org.hamcrest</package>
+                <package>org.hamcrest.core</package>
+                <package>org.hamcrest.internal</package>
                 <package>junit.extensions</package>
                 <package>junit.framework</package>
                 <package>junit.runner</package>
                 <package>junit.textui</package>
-                <package>org.hamcrest</package>
-                <package>org.hamcrest.core</package>
                 <package>org.junit</package>
+                <package>org.junit.experimental</package>
+                <package>org.junit.experimental.categories</package>
+                <package>org.junit.experimental.max</package>
+                <package>org.junit.experimental.results</package>
+                <package>org.junit.experimental.runners</package>
+                <package>org.junit.experimental.theories</package>
+                <package>org.junit.experimental.theories.internal</package>
+                <package>org.junit.experimental.theories.suppliers</package>
+                <package>org.junit.function</package>
+                <package>org.junit.internal</package>
```

Comment:
Why do you think the module should expose `internal` packages?

Reply:
My bad, I did not check the javadoc to see what packages should be exposed. I have rectified the packages that should be exposed as per the [javadoc](https://junit.org/junit4/javadoc/latest/index.html)